daemon/src/keystore.rs

```diff
@@ -0,0 +1,171 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
```

Comment:
Would we wanna peel this away not maintain keystore util in this crate? Not for this change but maybe something to capture.